src/seedu/addressbook/data/person/Address.java

```diff
@@ -38,22 +55,127 @@ public static boolean isValidAddress(String test) {
 
     @Override
     public String toString() {
-        return value;
+        String address ="";
+        address+=block+SPLIT_TOKEN;
+        address+=street+SPLIT_TOKEN;
+        address+=unit+SPLIT_TOKEN;
+        address+=postalCode;
+
+        return address;
```

Comment:
While this will work, it might be cleaner to consider something like: `return block + SPLIT_TOKEN + street + SPLIT_TOKEN + unit + SPLIT_TOKEN + postalCode`